tutorial/tutorial-editing-page-content-1.md

```diff
@@ -1,3 +1,3 @@
-### Creating a project directory
+##### Step 1
```

Comment:
Can we instead of just saying "step 1" say "step 1 - setup your project directory"? For all these steps?